### PR TITLE
Merging "Re-add a NETCore-targeted Compilers NuGet package" from master.

### PR DESCRIPTION
**Pull request by VSadov**

### CODE REVIEW

**Comment by VSadov** [comment]
@agocke - please review
@OmarTawfik - FYI

File: src/Compilers/CSharp/CscCore/CscCore.csproj [hunk 1]
```diff
@@ -17,7 +17,6 @@
     <ProjectTypeGuids>{786C830F-07A1-408B-BD7F-6EE04809D6DB};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
     <TargetFramework>netcoreapp1.1</TargetFramework>
     <RuntimeIdentifiers>win7-x64;ubuntu.14.04-x64;ubuntu.16.04-x64;osx.10.12-x64</RuntimeIdentifiers>
-    <PackageTargetFallback>portable-net452</PackageTargetFallback>
```
Comment (agocke) [inline src/Compilers/CSharp/CscCore/CscCore.csproj]: I'm not sure where this came from but it seems fine.

Comment (VSadov) [inline src/Compilers/CSharp/CscCore/CscCore.csproj]: I've just copied these proj files from master, so here is the diff. 
No idea where this came from in the branch.

**Comment by VSadov** [comment]
test windows_release_vs-integration_prtest please

**Comment by VSadov** [comment]
integration tests are flaky in the branch

**Pull request merged**